zhaquirks/inovelli/__init__.py

```diff
@@ -245,66 +193,240 @@ def handle_cluster_request(
             return
 
 
-VZM35SN_REMOVES = [
-    0x0012,
-    0x0013,
-    0x0014,
-    0x0019,
-    0x001A,
-    0x0034,
-    0x0064,
-    0x007D,
-    0x0105,
-]
+class InovelliVZM31SNCluster(InovelliCluster):
+    """Inovelli VZM31-SN custom cluster."""
+
+    cluster_id = 0xFC31
+    name = "InovelliVZM31SNCluster"
+    ep_attribute = "inovelli_vzm31sn_cluster"
+
+    attributes = {
+        key: InovelliCluster.attributes[key] for key in InovelliCluster.attributes
+    }
```

Comment:
Can you use:
```python
attributes = InovelliCluster.attributes.copy()
```
Please also test if you still have the initialization error then.